FrontDesk/src/FrontDesk.Core/Aggregates/Room.cs

```diff
@@ -6,10 +6,12 @@ namespace FrontDesk.Core.Aggregates
     public class Room : BaseEntity<int>, IAggregateRoot
     {
         public virtual string Name { get; private set; }
+        public string Color { get; private set; }
 
-        public Room(string name)
+        public Room(string name, string color)
```

Comment:
The core domain model doesn't need to know what the UI uses for the color of the room. It's not literally the room's color in real life.

Reply:
Done